Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 authors = ["Kisio Digital <team.coretools@kisio.org>", "Guillaume Pinot <texitoi@texitoi.eu>"]
 name = "transit_model"
-version = "0.10.0"
+version = "0.11.0"
```

Comment:
I don't think it's a good idea. If this PR is merged, in the next PR I can also put 0.12.0 and deploys whereas 0.10.0 has never been deployed.

Reply:
No, you wouldn't put `0.12.0` because you would check first what is the last released version. The idea is not to increment it everytime you do change something, the idea is just to verify that is coherent with 2 information:
- what is the last released version
- what kind of change my PR is doing, and keep or change the next version accordingly